messages/jsonschema/scripts/templates/java.java.erb

```diff
@@ -0,0 +1,96 @@
+<% @schemas.each do |key, schema| %>
+    public static class <%= class_name(key) %> {
+      <%- schema['properties'].each do |property_name, property| -%>
+        private <%= type_for(class_name(key), property_name, property) -%> <%= property_name %><%= property['items'] ? ' = new java.util.ArrayList<>()' : '' %>;
+      <%- end -%>
+
+        public <%= class_name(key) %>() {}
```

Comment:
Consider removing the default constructor.

Reply:
Only classes without required fields have a public empty constructor now.